apps/project-renderer/src/components/app/index.js

```diff
@@ -69,6 +69,15 @@ const App = ( {
 			data = {};
 		}
 
+		if ( preview ) {
```

Comment:
Actually, on a second look - this shouldn't be necessary at all. It's already enabled in `useEffect` above if `?preview` is set.

Reply:
That should be true, let me test that